pkg/models/config.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+// ModuleConfig is an interface to describe the config of a module, which is organized in the
```

Comment:
Whether ModuleConfig is an interface or a struct? The comment conflicts with the code

Reply:
It's a struct. This is a miswriting.